contribs/gmf/src/editing/Snapping.js

```diff
@@ -553,7 +553,18 @@ EditingSnappingService.prototype.loadItemFeatures_ = function(item) {
 
       // (3) Read features from request response and add them to the item
       const readFeatures = new olFormatWFS().readFeatures(response.data);
+      let me = this;
       if (readFeatures) {
+        if (item.snappingConfig.invertXY) {
+          readFeatures.forEach(function (f) {
```

Comment:
Perhaps you should use the syntax ``f => {`` instead of ``function (f) {``?

Reply:
The whole file uses function(f) and not arrow functions. I used this syntax for consistency